app/javascript/vue/repository_row/manage_stock_value_modal.vue

```diff
@@ -175,14 +175,21 @@
         return unit ? unit[1] : ''
       },
       newAmount: function() {
+        const amount = isNaN(this.amount) ? 0 : this.amount;
+        let value
         switch (this.operation) {
           case 2:
-            if (this.amount) return parseFloat(this.stockValue.amount) + this.amount
+            console.log([ parseFloat(this.stockValue.amount), amount])
+            value = parseFloat(this.stockValue.amount) + amount
+            break;
           case 3:
-            if(this.amount) return parseFloat(this.stockValue.amount) - this.amount
+            value = parseFloat(this.stockValue.amount) - amount
+            break;
           default:
-            return this.amount
+            value = amount
+            break;
         }
+        return Math.round(value * 10**parseInt(this.stockValue.decimals)) / 10**parseInt(this.stockValue.decimals)
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/EPSILON

Does this also need to be used here?

Reply:
@artoscinote 199.773 - 177.9 = 21.87299999999999 on the brower.